ik_control_parameters:
  position_threshold: 0.0007
  velocity_threshold: 0.0007
  hand_max_velocity: 2.0
  hand_position_threshold: 0.005
  # kinematics_only: true
  kinematics_only: false
  chain_group_names:
    - left_hand
    - right_hand
    # - left_arm
    # - right_arm
  tree_group_names:
    - both_hands
  group_map:
    left_hand: left_hand_arm
    right_hand: right_hand_arm
    both_hands: dual_hand_arm
    # right_arm: right_arm
    # left_arm: left_arm
  motion_planner:
    planner_id: RRTstarkConfigDefault
    planning_time: 1.0
    # planner_id: RRTConnectkConfigDefault
    # planning_time: 5.0
    goal_position_tolerance: 0.005
    goal_orientation_tolerance: 0.005
    goal_joint_tolerance: 0.005
    workspace_bounds: [-1.2, -1.5, 0.1, 0.2, 1.5, 1.5]